packages/document/main-doc/docs/en/guides/basic-features/routes.mdx

```diff
@@ -412,3 +412,17 @@ export default defineConfig({
   },
 });
 ```
+
+## FAQs
+
+1. The code in the product is es2015+ but expect to be in es5.
+
+The default product of react-router^6 is in es2020. If you need the code in the product to be in es5, you can configure `source.include` to get the relevant packages of react-router compiled by the bundler.
+
+```
```

Comment:
We can set the language of code block to js:

```suggestion
```js
```

Reply:
done